pkg/cmd/generate-bazel-extra/main.go

```diff
@@ -48,15 +49,54 @@ func runBuildozer(args []string) {
 	}
 }
 
-func getTestTargets(testTargetSize string) ([]string, error) {
-	if _, ok := testSizeToDefaultTimeout[testTargetSize]; !ok {
-		return nil, errors.New("testTargetSize should be one of {small,medium,large,enormous}")
+// parseQueryXML is used because Go doesn't support parsing XML "1.1".
+// It returns a map where the key is a test size {small,medium,large,enormous} and
+// the value is a list of test targets having that size.
+func parseQueryXML(data []byte) (map[string][]string, error) {
+	targetNameRegex, err := regexp.Compile(`<rule class="go_test" location=".*" name="(.*)">`)
+	if err != nil {
+		return nil, err
+	}
+
+	targetSizeRegex, err := regexp.Compile(`<string name="size" value="(.*)"/>`)
+	if err != nil {
+		return nil, err
+	}
+
+	sizeToTargets := map[string][]string{}
+	var currentTargetName string
+	for _, line := range strings.Split(string(data), "\n") {
+		// Check if the line contains a target name.
+		line = strings.TrimSpace(line)
+		submatch := targetNameRegex.FindStringSubmatch(line)
+		if submatch != nil {
```

Comment:
I think you can make this (and the next call to `FindStringSubmatch()`) one line like:

```
if submatch := targetNameRegex.FindStringSubmatch(line); submatch != nil {
    ...
}
```